src/app/code/community/Shopgate/Cloudapi/Model/Frontend/Observer/OnepageSuccessAction.php

```diff
@@ -45,8 +50,8 @@ public function execute(Varien_Event_Observer $observer)
                 'serial' => Shopgate_Cloudapi_Block_PipelineRequest::PIPELINE_REQUEST_SERIAL,
                 'name'   => Shopgate_Cloudapi_Block_PipelineRequest::PIPELINE_REQUEST_CREATE_NEW_CART_FOR_CUSTOMER,
                 'input'  => array(
-                    'orderId' => $newOrderId
-                )
+                    'orderId' => $newOrderId,
+                ),
             )
```

Comment:
Please remove the trailing commas. We are not using this style in this project. I also see the benefits of it, but we need to decide in the team first and afterwards change it for all arrays.

Reply:
Done